mms/tests/integration_tests/test_utils.py

```diff
@@ -44,6 +44,7 @@
     'resnext-101-64x4d': 'https://s3.amazonaws.com/model-server/models/resnext-101-64x4d/resnext-101-64x4d.model',
     'squeezenet_v1.1': 'https://s3.amazonaws.com/model-server/models/squeezenet_v1.1/squeezenet_v1.1.model',
     'vgg19': 'https://s3.amazonaws.com/model-server/models/vgg19/vgg19.model',
+    'ferplus': 'https://s3.amazonaws.com/model-server/models/FERPlus/FERPlus.model',
```

Comment:
This won't work. If the data input is `Input2505` for the input layer, the test utils only expects to see `data` or `input0`. I noticed this with K's keras blog - it used `/input11` or something. I thought the slash was a problem and the input layer deviation was super interesting. Really need to think through the implications of this for the zoo, mms, and how stuff is tested.

Reply:
Yes, note my comments above. The integration suite code needs to be refactored to be able and handle broader set of models.